internal/fs/inode/file.go

```diff
@@ -178,38 +180,74 @@ func (f *FileInode) clobbered(ctx context.Context) (b bool, err error) {
 	return
 }
 
-// Ensure that f.content != nil
+// Helper method to open a reader
+// (should only be called by methods that have locks)
```

Comment:
What locks? Is it `f.mu`? Then, it is already mentioned in line 186. This comment is redundant.